pylutron_caseta/smartbridge.py

```diff
@@ -663,6 +722,21 @@ def _process_occupancy_group(self, occgroup):
             name=f"{self.areas[occgroup_area_id]['name']} Occupancy",
         )
 
+    async def _subscribe_to_button_status(self):
+        """Subscribe to pico button status updates."""
+        _LOG.debug("Subscribing to pico button status updates")
+        self.buttons
+        try:
+            for button in self.buttons:
+                response, _ = await self._subscribe(
+                   "/button/"+str(button)+"/status/event", self._handle_button_status
+                )
+                _LOG.debug("Subscribed to device status")
```

Comment:
Should this log message be more specific about subscribing to status of a specific button?

Alternatively, if somebody has a lot of pico remotes they would get about 3x as many occurrences of this message in the debug log, right? Maybe this should be outside the loop.

Reply:
Feels like if you're debugging, having each one display would be helpful?  Adding button ID